siac/hostcmd.go

```diff
@@ -55,8 +56,7 @@ func hostconfigcmd(param, value string) {
 		// convert price to hastings/byte/block
 		p, ok := new(big.Rat).SetString(value)
 		if !ok {
-			fmt.Println("could not parse price")
-			return
+			dieNoError("Could not parse price")
```

Comment:
whoops, I was confused lol.
A better approach would be to make `die` variadic, like `Println`:
```go
func die(args ...interface{}) {
    fmt.Fprintln(os.Stderr, args...)
    os.Exit(exitCode)
}